_infra/helm/uaa/values.yaml

```diff
@@ -7,8 +7,9 @@ rollingUpdate:
   maxUnavailable: 1
 
 image:
-  name: "cloudfoundry/uaa"
-  version: "76.18.0"
+  name: europe-west2-docker.pkg.dev/ons-ci-rmrasbs/images/uaa
+  pullPolicy: Always
+  version: latest
```

Comment:
We should pin this explicitly even if we manage our own tagged version. Currently v78.4.0

Reply:
This has been updated.